date-mod.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+var myObj = (function iife(){
+  //sets the day of the month to the date object
+  function setDate(){
+    var sd = new Date();
```

Comment:
This function should take one parameter, and it should instantiate a date instance for your module. Currently your setDate method isn't taking any parameters (refer to the assignment instructions for the specifications describing what this method should do.) I also strongly recommend you watch the recording where I cover the Module_2 homework.